_posts/en/newsletters/2021-02-17-newsletter.md

```diff
@@ -0,0 +1,311 @@
+  [descriptor][topic descriptors] templates (including with the recently-added
+  `sortedmulti`), and more.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.12.1-beta.rc1][] is a release candidate for a maintenance
+  release of LND.  It fixes an edge case that could lead to accidental
+  channel closure and a bug that could cause some payments to fail
+  unnecessarily, plus some other minor improvements and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20944][] adds a new `total_fee` field to the object returned
+  by the `getmempoolinfo` RPC and the `mempool/info` REST endpoint. `total_fee`
+  indicates the sum of the transaction fees for all transactions currently in the
+  mempool.
+
+- [LND #4909][] adds new `getmccfg` and `setmccfg` RPCs that can,
+  respectively, retrieve and temporarily change settings in LND's
+  *mission control* subsystem without restarting the daemon.  Mission
+  control uses information about past payment attempts to choose better
+  routes for subsequent payment attempts.
+
+- [Rust-Lightning #787][] ensures that channel closures due to error messages
+  only happen when the peer who sent the message is the channel's counterparty.
+  Previously, it was possible for malicious peers to force-close any channel if
+  they knew the channel id.
+
+- [BTCPay Server #2164][] redesigns the wallet setup wizard, walking the user
+  through setting up BTCPay's internal wallet and optionally integrating
+  it with the user's own remote software or hardware wallet.  This is
+  the start of other redesign work on BTCPay's interface.
+
+- [HWI #443][] adds support for signing multisig inputs with the
+  BitBox02 hardware wallet.
+
+- [Bitcoin Core #19145][] extends the `gettxoutsetinfo` RPC's
+  `hash_type` option to accept a new `muhash` parameter that will
+  produce the [MuHash3072][] digest of the UTXO set at the current block
+  height.  This is an alternative to producing the default SHA256 digest of
+  the UTXO set.  When run this way, MuHash has to process the same
+  amount as data as the SHA256 function, so there's not expected to be
+  any significant change in performance of the `gettxoutsetinfo` RPC,
+  which can take up to several minutes to return on slow single-board
+  computers.  However, a previously-calculated MuHash digest can have
```

Comment:
I call the finalized `uint256` result of the MuHash object the 'digest' and the intermittent `MuHash3072` object 'muhash object'. The object is what can be updated quickly.

```suggestion
  computers.  However, a previously-calculated MuHash object can have
```

Reply:
Oh, that's a good distinction; I feel dumb now for treating those two obviously different things as the same.  Thanks!